DXRModules/DeusEx/Classes/DXRMissions.uc

```diff
@@ -1180,6 +1229,11 @@ function MoveGoalToLocation(Goal g, GoalLocation Loc)
             a.Event = '';
             a.Destroy();
         }
+
+        if (g.name=="DTS" && dxr.localURL == "06_HONGKONG_WANCHAI_STREET" && Loc.name!="Sword Case"){
```

Comment:
this is already inside of the if statement checking map name, I think there's some redundant checks here?